rover_video/src/video_recording.hpp

```diff
@@ -0,0 +1,95 @@
+#ifndef __VIDEO__RECORDING__
+#define __VIDEO__RECORDING__
+
+#include "rclcpp/rclcpp.hpp"
+#include "rovus_lib/macros.h"
+
+#include "opencv2/core.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/imgcodecs.hpp"
+
+#include <atomic>
+#include <chrono>
+#include <sstream>
+#include <thread>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+/**
+ * @brief Each Recording objects handle 1 camera stream \n
+ * @brief They must be initialized with startRecording()
+ * @attention Cameral URL must be RTSP stream for GStreamer pipeline
+ */
+class Recording
+{
+  private:
+    static constexpr uint8_t RECORDING_INTERVAL = 20U;  // in seconds
```

Comment:
When declaring mesurable constant you can use the unit in the name. For example:

```suggestion
    static constexpr uint8_t RECORDING_INTERVAL_S = 20U;
```